src/EnergyPlus/Fans.hh

```diff
@@ -199,8 +205,8 @@ namespace Fans {
               PLBeltEffReg1CurveIndex(0), PLBeltEffReg2CurveIndex(0), PLBeltEffReg3CurveIndex(0), MotorMaxEffCurveIndex(0), PLMotorEffCurveIndex(0),
               VFDEffCurveIndex(0), DeltaPressTot(0.0), FanAirPower(0.0), FanSpd(0.0), FanTrq(0.0), FanWheelEff(0.0), FanShaftPower(0.0),
               BeltMaxEff(0.0), BeltEff(0.0), BeltInputPower(0.0), MotorMaxEff(0.0), MotorInputPower(0.0), VFDEff(0.0), VFDInputPower(0.0),
-              MaxFanPowerEncountered(0.0), FlowFractSchedNum(0), AvailManagerMode(0), MinTempLimitSchedNum(0), BalancedFractSchedNum(0),
-              UnbalancedOutletMassFlowRate(0.0), BalancedOutletMassFlowRate(0.0), AirLoopNum(0), DesignPointFEI(0.0)
+              MaxFanPowerEncountered(0.0), FlowFractSchedNum(0), MinTempLimitSchedNum(0), BalancedFractSchedNum(0), UnbalancedOutletMassFlowRate(0.0),
```

Comment:
I was afraid Fans.cc and hh might be modified in a different branch so I tried to hold off from any deeper refactors here.